examples/eww-bar/eww.scss

```diff
@@ -31,24 +32,11 @@
   margin-left: 10px;
   margin-right: 20px;
 }
-.metric scale trough highlight {
-  all: unset;
-  background-color: #D35D6E;
-  color: #000000;
-  border-radius: 10px;
-}
```

Comment:
If you remove the highlight styles here, this _does_ break the UI

Reply:
i regret to inform you that it doesn't, both are duplicated.
as of ee937b90ec0b19d545dc4f415156cd375e0b0fa2, [lines 19-33](https://github.com/elkowar/eww/blob/ee937b90ec0b19d545dc4f415156cd375e0b0fa2/examples/eww-bar/eww.scss#L19-L33) are the exact same as [34-48](https://github.com/elkowar/eww/blob/ee937b90ec0b19d545dc4f415156cd375e0b0fa2/examples/eww-bar/eww.scss#L34-L48).